pan_firewall/assets/logs/pan.firewall.yaml

```diff
@@ -230,4 +385,1028 @@ pipeline:
+            - filter:
+                query: "@subtype:start"
+              name: "Open"
+            - filter:
+                query: "@subtype:end"
+              name: "Close"
+            - filter: "@subtype:drop"
+              name: "Reset"
+            - filter: "@subtype:deny"
+              name: "Refuse"
+          target: ocsf.activity_name
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'network.destination.ip' to 'ocsf.dst_endpoint.ip'
+          enabled: true
+          sources:
+            - network.destination.ip
+          sourceType: attribute
+          target: ocsf.dst_endpoint.ip
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'network.client.ip' to 'ocsf.src_endpoint.ip'
+          enabled: true
+          sources:
+            - network.client.ip
+          sourceType: attribute
+          target: ocsf.src_endpoint.ip
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'network.destination.port' to 'ocsf.dst_endpoint.port'
+          enabled: true
+          sources:
+            - network.destination.port
+          sourceType: attribute
+          target: ocsf.dst_endpoint.port
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'network.client.port' to 'ocsf.src_endpoint.port'
+          enabled: true
+          sources:
+            - network.client.port
+          sourceType: attribute
+          target: ocsf.src_endpoint.port
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: string-builder-processor
+          name: 99 - in attribute ocsf.connection_info.direction_id
+          enabled: true
+          template: "99"
+          target: ocsf.connection_info.direction_id
+          replaceMissing: false
+        - type: string-builder-processor
+          name: 0 - in attribute ocsf.connection_info.direction_id
+          enabled: true
+          template: "0"
+          target: ocsf.connection_info.direction_id
+          replaceMissing: false
+        - type: attribute-remapper
+          name: Remap `sessionid` to `ocsf.connection_info.session.uid`
+          enabled: true
+          sources:
+            - sessionid
+          sourceType: attribute
+          target: ocsf.connection_info.session.uid
+          targetType: attribute
+          targetFormat: string
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'app' to 'ocsf.connection_info.protocol_name'
+          enabled: true
+          sources:
+            - proto
```

Comment:
Is it `app` or `proto`?

Again the linter should have flagged this, needs to be brought up with logs.